admin_guide/managing_pods.adoc

```diff
@@ -146,16 +146,126 @@ endif::openshift-enterprise,openshift-origin[]
 
 [[admin-guide-controlling-egress-traffic]]
 == Controlling Egress Traffic
-As an {product-title} cluster administrator, you can control egress traffic
-using either a xref:admin-guide-deploying-an-egress-router-service[router] or
-xref:admin-guide-limit-pod-access-egress[firewall] methods.
+As an {product-title} cluster administrator, you can control egress traffic in two ways:
+
+xref:admin-guide-limit-pod-access-egress[Firewall]::
+Using an egress firewall allows you to enforce the acceptable outbound traffic
+policies, so that specific endpoints or IP ranges (subnets) are the only
+acceptable targets for the dynamic endpoints (pods within {product-title}) to
+talk to.
+
+xref:admin-guide-limit-pod-access-egress-router[Router]::
+Using an egress router allows you to create identifiable services to send
+traffic to a specific destination, ensuring an external source treats traffic as
+though it were coming from a known destination. This helps with security,
```

Comment:
...traffic to a specific destination, ensuring an external **destination** treats traffic as though it were coming from a known **source**.